src/cloudWatchLogs/commands/viewLogStream.ts

```diff
@@ -0,0 +1,158 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as nls from 'vscode-nls'
+import * as moment from 'moment'
+const localize = nls.loadMessageBundle()
```

Comment:
Why is this hanging out up here amongst the imports?

Reply:
I think the better question is why `moment` is in there, actually. Traditionally we import `vscode-nls` and initialize `localize` one after the other, above all the other import statements. Will move.